src/partials/head-meta.hbs

```diff
@@ -12,8 +14,14 @@
     {{/with}}
 
     {{!-- Swiftype - TODO: to remove --}}
+    {{#if (eq "docs" page.attributes.theme)}}
```

Comment:
suggestion: What about using an attribute similarly to what we are doing for the post type?

You can already define the following in your playbook or component:

```yml
asciidoc:
  attributes:
    page-type: 'Docs'
```

We could introduce a `page-doc-site`?
For the record, I'm not a big fan of `page-type`, I would rather use explicit attribute in the case. We don't necessarily need to mention swiftype but in my opinion something more specific would be better:

```yml
asciidoc:
  attributes:
    page-search-type: 'Docs'
    page-search-site: 'Reference Docs'
    # page-search-index: 'Reference Docs'
```

What do you think?

Reply:
Attributes are good I guess.

Aside: will the move to Algolia mean we won't need this anyway?

Reply:
if Algolia happens to use the same value then we can keep the same definition and only update this template. If Algolia is using other values, then we will both need to update the attribute definitions and the template.

Reply:
Updating as discussed here and on Slack